apps/frontend/src/app/3_organisms/FastBtcDialog/config/contracts.ts

```diff
@@ -0,0 +1,38 @@
+import { ethers } from 'ethers';
+
+import { Chains } from '../../../../config/chains';
+import { Environments } from '../../../../types/global';
+import erc20TokenAbi from './abi/erc20.json';
+
+export type FastBTCWithdrawContractName = 'btcToken' | 'btcWrapperToken';
+
+export type AppModelToContractAddressMap = Record<Environments, string>;
+
+export type ChainToContractsMap = {
+  address: Partial<Record<string, AppModelToContractAddressMap>>;
+  abi: ethers.ContractInterface;
+};
+
+export const contracts: Record<
+  FastBTCWithdrawContractName,
+  ChainToContractsMap
+> = {
+  btcToken: {
+    address: {
+      [Chains.RSK]: {
+        [Environments.Mainnet]: ethers.constants.AddressZero,
+        [Environments.Testnet]: ethers.constants.AddressZero,
+      },
+    },
+    abi: erc20TokenAbi,
+  },
+  btcWrapperToken: {
+    address: {
+      [Chains.RSK]: {
+        [Environments.Mainnet]: '0xa233108b33dc77f1eee9d183ee1dc9725e76d475',
+        [Environments.Testnet]: '0xf629e5c7527ac7bc9ce26bdd6d66f0eb955ef3b2',
+      },
+    },
+    abi: erc20TokenAbi,
+  },
+};
```

Comment:
You can move these to the `@sovryn/contracts` in the `tokens` folder. erc20 file then could be deleted.
As it's only RSK network you could get contract by `getTokenContract('btcWrapperToken', getRskChainId())`